packages/api/index.js

```diff
@@ -28,6 +28,7 @@ export async function init(config = {}) {
 
 export async function shutdown() {
   if (actualApp) {
+    await actualApp.send('sync');
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I don't think this is a good idea. When I try to close an app and it does not close within 3 seconds - I manually "force close" it. And I'm guessing it could corrupt the database for Actual then. If the sync is taking a while, this would be problematic.

Reply:
This is specifically for the API not the user-facing app. Do you think the same user behavior would be present in API client based projects?

Reply:
Ah! No, for API this is fine